### ISSUE TITLE
LinearChart left right padding

### ISSUE THREAD

**Issue by r0otx** [danielgindi/Charts#5113]
Hello, thank you for this beautiful package.

I have problem, my chart half hided and I need add padding for chart. How I can make it?

<img width="282" alt="image" src="https://github.com/danielgindi/Charts/assets/38701624/39327776-823a-4321-92e9-da44468be29d">

**Comment by ygdhsb1995** [comment]
chart.setExtraOffsets(left: 0, top: 0, right: 20, bottom: 0)

**Comment by r0otx** [comment]
> chart.setExtraOffsets(left: 0, top: 0, right: 20, bottom: 0)

No, it's moved all chart layer, but I'm need padding left and right only data line
